<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>iDEC NEFU_China 2021</title>
    <link rel="stylesheet" type="text/css" href="../../CSS/main_cssstyle.css">
	<style>
		header{
		    background: url("../../resources/header1.png");
		    min-height: 450px;
		    background-repeat: no-repeat;
		    background-size: cover;
		}
		.page-title {
		    padding-top:10em;
		    padding-bottom: 1.5em;
		}
		/* footer的样式 */
		.contact-icon{
		    display: flex;
		    align-items: center;
		    /*font-size: 3em;*/
		    color: white;
		}
		.contact-icon a{
		    color: white;
		}
		.contact-icon i{
		    font-size: 3em;
		}
	</style>
</head>
<body>
<!--加载动画-->
<div class="loader-wrapper">
    <span class="loader"><img src="../../resources/Loading.gif" alt="Our animated logo is keeping you company until the page has loaded.">
    </span>
</div>

<a class="skip-main" href="#slide1-chapter1">Skip to main content</a>
<header>
    <nav id="normal-nav"> <!---- 常规导航栏Navigation bar (normal) -->
        <div class="logo">
            <img style="max-height: 22em" id="nav-logo" src="../../resources/HMAlogo.png" alt="This is our logo, but if you click it, you will return home">
        </div>
        <div class="chapters">
            <div class="dropdown-menu cool-link">
                <a class="chapter-nav " href="https://idec2021.github.io/NEFU_China/">
                    <div class="chapter-content">
                        <div class="chapter-title active ">
                            Home
                        </div>
                    </div>
                </a>
            </div>
            <div class="dropdown-menu cool-link">
                <a class="chapter-nav " href="#">
                    <div class="chapter-content">
                        <div class="chapter-title active ">
                            Team
                        </div>
                        <div class="chapter-icon">
                            <i class="fas fa-sort-down"></i>
                        </div>
                    </div>
                </a>
                <div class="dropdown-content">
                    <a href="../TEAM/Members.html" class="dropdown-page cool-link2">
                        Members
                    </a>
                    <a href="../TEAM/Attributions.html" class="dropdown-page cool-link2">
                        Attributions
                    </a>
                </div>
            </div>
            <div class="dropdown-menu cool-link">
                <a class="chapter-nav " href="#">
                    <div class="chapter-content">
                        <div class="chapter-title active ">
                            Project
                        </div>
                        <div class="chapter-icon">
                            <i class="fas fa-sort-down"></i>
                        </div>
                    </div>
                </a>
                <div class="dropdown-content">
                    <a href="../PROJECT/Description.html" class="dropdown-page cool-link2">
                        Description
                    </a>
                    <a href="../PROJECT/Design.html" class="dropdown-page cool-link2">
                        Design
                    </a>
                    <a href="../PROJECT/Results.html" class="dropdown-page cool-link2">
                        Results
                    </a>
                </div>
            </div>
            <div class="dropdown-menu cool-link">
                <a class="chapter-nav " href="#">
                    <div class="chapter-content">
                        <div class="chapter-title active ">
                            Documentation
                        </div>
                        <div class="chapter-icon">
                            <i class="fas fa-sort-down"></i>
                        </div>
                    </div>
                </a>
                <div class="dropdown-content">
                    <a href="../DOCUMENTATION/Experiments.html" class="dropdown-page cool-link2">
                        Experiments
                    </a>
                    <a href="../DOCUMENTATION/Notebook.html" class="dropdown-page cool-link2">
                        Notebook
                    </a>
                    <a href="../DOCUMENTATION/Safety.html" class="dropdown-page cool-link2">
                        Safety
                    </a>
                </div>
            </div>
        </div>
    </nav>
    <!-------- 侧边栏Navigation for mobile-------->
    <nav id="mobile-nav">
        <div class="logo">
            <a href="https://idec2021.github.io/NEFU_China/">
                <img style="max-height: 400px" id="nav-logo" src="../../resources/HMAlogo.png" alt="This is our logo, but if you click it, you will return home">
            </a>
        </div>
        <div class="mobile-nav">
            <span class="hamburger" onclick="openNav()"> <i class="fas fa-bars"></i> </span>
        </div>
        <div id="side-nav" class="mobile-sidenav">
            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
            <div class="mobile-dropdown-menu">
                <a href=""> Home </a>
            </div>
            <ul class="mobile-dropdown-menu">
                <li> <a> Team </a>
                    <ul class="mobile-dropdown-content">
                        <li>
                            <a href="../TEAM/Members.html" class="dropdown-page cool-link2"> Members </a>
                        </li>
                        <li>
                            <a href="../TEAM/Attributions.html" class="dropdown-page cool-link2"> Attributions </a>
                        </li>
                    </ul>
                </li>
            </ul>
            <ul class="mobile-dropdown-menu">
                <li> <a> Project  </a>
                    <ul class="mobile-dropdown-content">
                        <li>
                            <a href="../PROJECT/Description.html" class="dropdown-page cool-link2"> Description </a>
                        </li>
                        <li>
                            <a href="../PROJECT/Design.html" class="dropdown-page cool-link2"> Design </a>
                        </li>
                        <li>
                            <a href="../PROJECT/Results.html" class="dropdown-page cool-link2"> Results </a>
                        </li>
                    </ul>
                </li>
            </ul>
            <ul class="mobile-dropdown-menu">
                <li> <a> Documentation </a>
                    <ul class="mobile-dropdown-content">
                        <li>
                            <a href="../DOCUMENTATION/Experiments.html" class="dropdown-page cool-link2"> Experiments </a>
                        </li>
                        <li>
                            <a href="../DOCUMENTATION/Notebook.html" class="dropdown-page cool-link2"> Notebook </a>
                        </li>
                        <li>
                            <a href="../DOCUMENTATION/Safety.html" class="dropdown-page cool-link2"> Notebook </a>
                        </li>
                    </ul>
                </li>
            </ul>
        </div>
    </nav>
    <!------------ 页面标题PAGE TITLE -------------------->
    <div class="page-title">
        <div class="page-title-content">
            <h1>EXPERITIONS</h1><!---Page title -->
        </div>
    </div>
</header>
<div class="grid-normal">
    <!--------------Introduction/Summary--------------------->
    <div class="introduction grid-normal">
        <!-- <div class="introduction-content">
            <p>
                简介内容
            </p>
        </div> -->
    </div>
</div>
<div style="display:none" id="marker" class= "triangle-down"> </div>
<!------------------ BUTTONS ---------------------------->
<div class="buttons">
</div>
<div class="sections-container">
    <aside class="sticky">
        <div id="sub-nav"class="sub-navigation-content">
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter1">General</a><!------Chapter one -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter2">4HB biosensor construction<p></a><!------Chapter Two -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter3">Construction of PobR mutant library</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter4">Fluorescence assay and screening of the PobR mutant library</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter5">Ligand specificity test of selected PobR mutants</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter6">Site-directed mutagenesis of the PobR CDS</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter7">Modeling and docking</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter8">Adaptive laboratory evolution procedure</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter9">HMA determination</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter10">References</a><!------Chapter Three -->
            </div>

            <!--            返回顶部箭头-->
            <div  id="to-top-arrow">
                <a title="Back to top"href="#"><i class="fas fa-chevron-circle-up"></i></a><!------Chapter Three -->
            </div>
        </div>
    </aside>
    <div class="carousel">
        <div class="slider">
            <!------------------------------ section 1------------------------------------>
            <section class="slide-sections">
                <div class="main-body">
                    <div class="main-body-content">
                        <div id="slide1-chapter1">
                            <h2>General</h2>
                            <p><a href="../../resources/Experiments/transformation-bytupwnw.pdf" class="hyperlink">Transformation</a></p>
                            <p><a href="../../resources/Experiments/plasmid-dna-extraction-byqzpvx6.pdf" class="hyperlink">Plasmid DNA extraction</a></p>
                            <p><a href="../../resources/Experiments/polymerase-chain-reaction-pcr-bysrpwd6.pdf" class="hyperlink">PCR</a></p>
                            <p><a href="../../resources/Experiments/overlap-amp-gibson-ligation-byrspv6e.pdf" class="hyperlink">Overlap ligation & Gibson ligation</a></p>
                            <p><a href="../../resources/Experiments/transformation-bytupwnw.pdf" class="hyperlink">Transformation</a></p>
                            <p><a href="../../resources/Experiments/restriction-endonuclease-reactions-byutpwwn.pdf" class="hyperlink">Restriction endonuclease reactions</a></p>
                            <p><a href="../../resources/Experiments/nucleic-acid-amp-protein-electrophoresis-byu4pwyw.pdf" class="hyperlink">Nucleic acid & protein electrophoresis</a></p>
                            <p><a href="../../resources/Experiments/preparation-of-electrocompetent-cells-byuxpwxn.pdf" class="hyperlink">Preparation of electrocompetent cells</a></p>
                        </div>
                        <div id="slide1-chapter2">
                            <h2>4HB biosensor construction</h2>
                            <p>We constructed a 4HB biosensor based on plasmid pYB1a-PobR-eGFP-Cmr. This plasmid mainly contains two parts:</p>
                            <p>1. The <i>pobR</i> coding sequence (CDS) which can express PobR protein;</p>
                            <p>2. An engineered operon consisting of the enhanced green fluorescent protein (eGFP) and the chloramphenicol resistance (<i>Cmr</i>) genes.</p>
                            <p>To further understand each part, please <a href="../PROJECT/Design.html" class="hyperlink">click here</a> to view our design part.</p>
                            <p>To constructed pYB1a-PobR-eGFP-Cmr, we first tried to generate plasmid pYB1a-PobR-eGFP by Gibson assembly. PobR CDS was synthesized by Ruibiotech (Harbin, China) and amplified by PCR using the primers PobR-Gibson-F and PobR-Gibson-R. At the same time, plasmid pYB1a-eGFP was linearized using primers pYB1a-Gibson-F and pYB1a-Gibson-R. Then the two fragments were ligated using Gibson assembly and transformed into <i>E. coli</i> DH5α competent cells and the plasmid pYB1a-PobR-eGFP was constructed. <a href="../../resources/Experiments/overlap-amp-gibson-ligation-byrspv6e.pdf" class="hyperlink">Click here</a> to view our ligation protocol.</p>
                            <p>Secondly, we tried to insert Cmr gene into pYB1a-PobR-eGFP by using enzyme-cut and Link up. The Cmr CDS contained with the restriction endonuclease cleavage sites of XhoI and BglII was amplified by Cmr-F and Cmr-R primers using the Cmr CDS as a template. Then the fragment and the plasmid were digested with XhoI and BglII. Finally, we used T4 ligase (Thermo Fisher Scientific) to ligate the two fragments and transformed the ligation product into <i>E. coli</i> DH5α competent cells.</p>
                            <p>The result of DNA sequencing showed that our pYB1a-PobR-eGFP-Cmr plasmid was constructed.</p>
                        </div>
                        <div id="slide1-chapter3">
                            <h2>Construction of PobR mutant library</h2>
                            <p>Aiming to get the HMA biosensor with the best performance, a large PobR mutants library which is able to include as many mutation situations as possible is crucial. Therefore, we designed our mutants library construction method based on error-prone PCR and MEGWHOP ( megaprimer PCR of whole plasmid) cloning <a linkedidchapter="r1" class="main-nav-link hyperlink"><sup>[1]</sup></a>. This method has high efficiency and high positive fate compared with traditional methods. Please <a href="../../resources/Experiments/construction-of-mutant-library-byu8pwzw.pdf" class="hyperlink">click here</a> to view our detailed protocol. </p>
                            <p>A large library for stochastic mutagenesis of PobR was established by using error-prone PCR and MEGAWHOP (megaprimer PCR of whole plasmid). Mutations on PobR were performed by random mutagenesis kit (Solarbio Life Science). Random mutation codons were introduced into the amplified targeted <i>pobR</i> gene, due to Taq DNA polymerase lack 3’→5’ proofreading function in the reaction buffer system. It is conspicuous to substantiate that the positions mutated for the library were diversiform, and the mutated rate is 3‰on average by DNA sequencing. The amplified targeted <i>pobR</i> genes were inserted into the template by MEGAWHOP without requiring restriction digestion in these constructs <a linkedidchapter="r1" class="main-nav-link hyperlink"><sup>[1]</sup></a>. The enzyme of DpnI (New England Biolabs) was applied into the whole-plasmid PCR products to digest methylated and hemi methylated DNA templets <a linkedidchapter="r2" class="main-nav-link hyperlink"><sup>[2]</sup></a>.T4 DNA ligase (Thermo Fisher Scientific) was be employed to concatenate incision plasmid for improving the efficiency of transformation. Upon ligation with T4 DNA ligase, PobR<sup>mut</sup> constructs was obtained. The ligated products were transformed in DH5α competent cell (Tsingke Biotechnology).</p>
                            <p>Based on the methods introduced above, about 552,750 colonies containing PobR mutagenesis were established, in this PobR<sup>mut</sup> constructed library.</p>
								<div class="figure-container auto-margin">
								    <img src="../../resources/Experiments/figure1.png" alt="" style="width: 80%">
                                <p><strong>Figure1.</strong>1100 colonies grew on the plate after the plasmid with mutated points (20 ng) was transformed in DH5α competent cell (100 μl). 55 colonies were recuperated from per nanogram of DNA.</p>
                            </div>

                        </div>
                        <div id="slide1-chapter4">
                            <h2>Fluorescence assay and screening of the PobR mutant library</h2>
                            <p>Another part which is essential to get the best HMA biosensor is the screening part. To quickly and precisely selected clones from our huge mutant library, we used two different methods: chloramphenicol growth screening and fluorescence assay screening to meet our target. </p>
                            <p>First, we transformed the PobR mutant library into DH5α competent cells and cultured overnight in LB agar plates with 20 μg/mL chloramphenicol as growth pressure and 0.06 g/L HMA as inducer (50 μg/mL ampicillin was also added in LB to remain the plasmid).</p>
                            <p>In this progress, 639 clones grew on the plates were picked to our second step screening.</p>
                            <p>The second step of our screening is fluorescence assay. We used Tecan Spark<sup>®</sup> multimode microplate reader to measure OD<sub>600</sub> and fluorescence intensity of enhanced green fluorescence protein (eGFP). By adding 1 g/L HMA as inducer, the clones with higher induction of HMA would show a higher RFU/OD<sub>600</sub> value compared with blank control (no HMA was added) and other clones.</p>
                            <p>We used deep-well 96-well microplates containing 200 μl of LB medium with ampicillin to culture the clones picked from the first step. After shaking at 200 rpm and 37°C for 12h, 2 μl of the bacteria from each well was inoculated into 200 μl of LB medium with ampicillin and 1 g/L of HMA in deep-well 96-well microplates, and cultured on a shaking incubator for another 12 h. Finally, we picked 100 μl of the cultured bacteria from each well into the 96-well microplate to detect the OD<sub>600</sub> and green fluorescence (with 430 nm as the excitation wavelength and 510 nm as the emission wavelength). Please <a href="../../resources/Experiments/microplate-reader-operating-procedure-bysspwee.pdf" class="hyperlink">click here</a> to see our microplate reader operating procedure.</p>
                        </div>
                        <div id="slide1-chapter5">
                            <h2>Ligand specificity test of selected PobR mutants</h2>
                            <p>We used fluorescence assay to detect the ligand specificity of our PobR mutants. Six HMA analogues were used to induce our PobR mutants. The six analogues are MA (Mandelic Acid), HPP (4-Hydroxyphenylpyruvate), PPA (Phenylpyruvate), Trp (Tryptophan), Phe (Phenylalanine), 4HB (4-hydroxybenzoic acid). To study the induction of these analogues to the mutants, we first configurate their mother liquors with the concentration of 50 g/L and pH of 7. Subsequent steps were just like our fluorescence assay screening of the PobR mutant library. We cultured the mutants in 200 μl of LB medium with ampicillin for 12 h at 37 °C. Then, 2 μl of each clone was added to 200 μl of LB medium with ampicillin and 1 g/L of HMA, or each of its analogs (added 4 μl 50 g/L mother liquors), in the wells of a deep-well 96-well microplate, and cultured for 12 h at 37 °C in a shaking incubator. We picked 100 μl of the cultured bacteria from each well into the 96-well microplate to detect the OD<sub>600</sub> and green fluorescence (with 430 nm as the excitation wavelength and 510 nm as the emission wavelength). Finally, we calculated the RFU/OD<sub>600</sub> of each analogue to each mutant to estimate the mutants’ ligand specificity. Please <a href="../../resources/Experiments/microplate-reader-operating-procedure-bysspwee.pdf" class="hyperlink">click here</a> to see our microplate reader operating procedure.</p>
                        </div>
                        <div id="slide1-chapter6">
                            <h2>Site-directed mutagenesis of the PobR CDS</h2>
                            <p>To get the PobR mutants with single amino acid mutation, we designed pairs primers with mutation site present in the forward primers to having inverse PCR using pYB1a-PobR-eGFP-Cmr as template <a linkedidchapter="r3" class="main-nav-link hyperlink"><sup>[3]</sup></a>. After DNA purification, the original templates were eliminated by DpnI digestion.
                                Finally, the plasmids with nicks on them were transformed into <i>E. coli</i> DH5α competent cells to amplify the plasmids and repair the nicks.
                            </p>
                        </div>
                        <div id="slide1-chapter7">
                            <h2>Modeling and docking</h2>
                            <p>To better understand the effects of amino acid substitutions on the response of the PobR protein, we tried to use software to simulate the structure of the protein and construct molecular docking between PobR protein and its inducer.</p>
                            <p>We first simulated the structure of the PobR monomer by using AlphaFold2 <a linkedidchapter="r4" class="main-nav-link hyperlink"><sup>[4]</sup></a> and selected the predicted structure with the highest score (pLDDT = 91.4) from five predictive models which were generated by AlphaFold2 based on the confidence data to have our subsequent molecular docking.</p>
                            <p>Secondly, we obtained the structure files of the ligand 4HB from the organic small molecule database Pubchem (<a href="https://pubchem.ncbi.nlm.nih.gov/" class="hyperlink">https://pubchem.ncbi.nlm.nih.gov/</a>). </p>
                            <p>Based on the structures of PobR monomer and 4HB, molecular docking was simulated by docking engine Autodock Vina <a linkedidchapter="r5" class="main-nav-link hyperlink"><sup>[5]</sup></a>. Vina search space coordinates were set as center_x = -0.83, center_y = -1.292, center_z = 7.229. Dimensions of the search space were set as size_x = 33.75, size_y = 37.5, size_z = 37.5, and exhaustiveness was set at 10. The top 9 conformational conditions in a score based on the lowest binding energy (-6.0 kcal/mol) were listed as the docking results. To examine the accuracy of our docking, we used Ligplus to check the interaction between the small molecule and predicted protein receptors <a linkedidchapter="r6" class="main-nav-link hyperlink"><sup>[6]</sup></a>. Finally, the three-dimensional schematics of the protein and its ligand were portrayed using PyMol Version 2.2.0.</p>
                        </div>
                        <div id="slide1-chapter8">
                            <h2>Adaptive laboratory evolution procedure</h2>
                            <p>To evolve strains with higher HMA production three modules includes HMA biosensor module (plasmid pYB1a-PobR-eGFP-Cmr), HMA biosynthesis module (plasmid pRB1s-hmaS-aroG-pheA), mutagenesis module (plasmid pLT1k-MP6) were transformed into <i>E. coli</i> BL21 competent cells and named as HMA-0. Then, we inoculated 200 μL of M9 medium with 2% volume of <i>E. coli</i>, 2 g/L glucose and different concentrations of chloramphenicol (5, 10, 15, 20, 30, 40, 50 µg/mL) and shaking at 37 ℃ for 12 h.</p>
                            <p>Then, we stored the bacteria growth under the pressure of 50 µg/mL chloramphenicol and named as HMA-C50-1. This process was repeated using HMA-C50-1 under the pressure of 60, 80, 100, 120, 140, 160 µg/mL chloramphenicol. The HMA-2 was selected under the pressure of 60 µg/mL chloramphenicol. </p>
                            <p>Then, we gradually raised the chloramphenicol concentration to enrich HMA-producing cells but to eliminate other cells. When strains grow to stationary phase, 2 μL culture was stored and diluted into a fresh medium with same pressure or higher pressure of chloramphenicol compared with the round before, and the enrichment procedure was repeated for eight rounds. During the enrichment procedure, chloramphenicol was added at a concentration of 50, 60, 60, 65, 65, 70, 75 and 80 µg/mL. </p>
                        </div>
                        <div id="slide1-chapter9">
                            <h2>HMA determination</h2>
                            <p>We used HPLC (high performance liquid chromatography) as our HMA determination method.
                                We first activate the cells by inoculate 5 mL of LB medium with 1% volume of <i>E. coli</i> culture and shaking at 200 rpm and 37℃ for 12h. Then we inducted the expression of the protein on HMA biosynthesis module using ZYM-5052 medium and we also used fermentation medium M9 to help the cells produce HMA.</p>
                            <p>Finally, the HMA was measured an HPLC (Agilent 1260) equipped with a UV detector and a Welch Xtimate C8 column (4.6 × 250 mm, 3.5 µm). The samples were analyzed under the following gradient of eluent A (8 mM KH<sub>2</sub>PO<sub>4</sub>, pH 2.4) and eluent B (100% acetonitril): 5 min 10% B, 20 min linear gradient to 30% B, 3 min linear gradient to 50% B, 5 min 50% B, 2 min linear gradient to 10% B, 5 min 10% B. HMA was detected at 215 nm. Please <a href="../../resources/Experiments/hma-determination-byuupwww.pdf" class="hyperlink">click here</a> to see our detailed HMA determination protocol.</p>
                            <p>Please <a href="../../resources/Experiments/strains-plasmids-and-primers.pdf" class="hyperlink">click here</a> to see our strains, plasmids and primers used in our experiments.</p>
                        </div>
                        <div class="references" id="slide1-chapter10">
                            <h2>References</h2>
                            <p>
                            <ol>
                                <li id="r1">Miyazaki K: MEGAWHOP cloning: a method of creating random mutagenesis libraries via megaprimer PCR of whole plasmids. Methods Enzymol 2011, 498:399-406.</li>
                                <li id="r2">McClelland M, Nelson M, Raschke E: Effect of site-specific modification on restriction endonucleases and DNA modification methyltransferases. Nucleic Acids Research 1994, 22:3640-3659.</li>
                                <li id="r3">Liu H, Naismith JH: An efficient one-step site-directed deletion, insertion, single and multiple-site plasmid mutagenesis protocol. BMC Biotechnol 2008, 8:91.</li>
                                <li id="r4">Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, Tunyasuvunakool K, Bates R, Žídek A, Potapenko A, et al: Highly accurate protein structure prediction with AlphaFold. Nature 2021, 596:583-589.</li>
                                <li id="r5">Trott O, Olson AJ: AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. J Comput Chem 2010, 31:455-461.</li>
                                <li id="r6">Laskowski RA, Swindells MB: LigPlot+: multiple ligand-protein interaction diagrams for drug discovery. J Chem Inf Model 2011, 51:2778-2786.</li>
                            </ol>
                            </p>
							<p></p><p></p><p></p>
                        </div>
                    </div>
                </div>
            </section>
        </div>
    </div>
</div>
<a href="#" id="mobile-to-top-arrow" class="to-top">
    <p style="font-size: 10px;text-align: center">Top of</p>
    <p style="font-size: 10px;text-align: center">the page</p>
    <img style="max-height: 70px" src="../../resources/to_top.png">
</a>
<!---------- FOOTER ------------>
    <footer id="footer">
        <div class="contact-info">
            <div class="contact-info-content">
                <!--- About Us-->
                <div class="box">
                    <h3>
                        About Us
                    </h3>
                    <p>
                        This team is composed of enthusiastic  students from three different colleges and various majors. To spy out the secrets of directed evolution, a group of beginners are looking forward to creating extraordinary experiences and we did it.
                    </p>
                </div>
                <!---- Follow Us -->
                <div class="box">
                    <h3>
                        Helpful links
                    </h3>
                    <h4><a target="_blank" href="../../resources/poster.jpg">Poster</a></h4>
                    <h4><a target="_blank" href="../../page/PROJECT/Design.html">Design</a></h4>
                    <h4><a target="_blank" href="../../page/DOCUMENTATION/Notebook.html">Notebook</a></h4>
                </div>
                <!---- Contact Us-->
                <div class="box">
                    <h3>
                        Address
                    </h3>
                    <p>
                        Northeast Forestry University, Harbin, Heilongjiang 150040, P. R. China
                    </p>
                </div>
            </div>
            <hr color="white">
            <div class="contact-info-content">
                <div class="contact-icon">
                    <a target="_blank" href=""> <i class="fa fa-envelope"></i>IDEC_NEFU2021@163.com</a>
                </div>
                <div class="contact-icon">
                    <a target="_blank" href=""> <i class="fab fa-facebook-square"></i>iDEC NEFU China</a>
                </div>
                <div class="contact-icon">
                    <a target="_blank" href=""> <i class="fab fa-twitter-square"></i>iDEC NEFU</a>
                </div>
            </div>
        </div>
    </footer>


<script src= "../../JS/jQuery.js" type =text/javascript></script>
<script src="../../JS/detect.js" type= "text/javascript"></script>
<script src="../../JS/main_javascript.js" type =text/javascript></script>
<script src="../../JS/MoveContent.js" type= "text/javascript"></script>
<script src="https://kit.fontawesome.com/50c1a3d91f.js" crossorigin="anonymous"></script>

<script>
    document.addEventListener("DOMContentLoaded", function(event) {
        MoveContentOutside();
    });
</script>
</body>
</html>
